draft-ietf-ace-edhoc-oscore-profile.md

```diff
@@ -515,23 +508,17 @@ That is, C sends a POST request to the /authz-info endpoint at RS, with the requ
-
-1. C may be required to re-POST the access token, since RS may have deleted a stored access token (and associated OSCORE Security Context) at any time, for example due to all storage space being consumed. This situation can be detected by C when it receives a 4.01 (Unauthorized) response from RS, especially as an "AS Request Creation Hints" message (see {{Section 5.3 of RFC9200}}.
+The Client provisioning an initial access token to the RS is followed by the execution of the EDHOC protocol (or combined using EAD as described in {{edhoc-exec}}) and by the derivation of an OSCORE Security Context, as detailed later in this section.
 
-2. C may be posting the first access token in a new series, e.g., because the old access token expired and thus its series terminated.
+The same procedure applies in other cases without a valid OSCORE Security Context shared between the C and RS, for example:
 
-3. C may need to update the OSCORE Security Context, e.g., due to a policy limiting its use in terms of time or amount of processed data, or to the imminent exhaustion of the OSCORE Sender Sequence Number space. The OSCORE Security Context can be updated by:
+* The old access token has expired (and thus its series terminated and associated OSCORE Security Context deleted) so C provisions a new access token to RS.
 
-    a. using the KUDOS key update protocol specified in {{I-D.ietf-core-oscore-key-update}}, if supported by both C and RS; or
+* The Client re-POSTs an access token which was deleted by RS (and thus its associated OSCORE Security Context) for example due to lack of storage. This situation can be detected by C when it receives a 4.01 (Unauthorized) response from RS, e.g., as an "AS Request Creation Hints" message, see {{Section 5.3 of RFC9200}}.
 
-    b. re-posting the access token using the same procedure as in case 1 above.
+Another case is if there is a valid OSCORE Security Context but it needs to be updated, e.g., due to a policy limiting its use in terms of time or amount of processed data, or to the imminent exhaustion of the OSCORE Sender Sequence Number space.
 
-In cases 1, 2 and 3b, C and RS rely on a protocol similar to the `coap_oscore` profile {{RFC9203}}, but leveraging the EDHOC-KeyUpdate function (see {{edhoc-key-update}}) before deriving a new OSCORE Security Context.
-
-Case 3a provides a lightweight alternative independent of ACE, and does not require the posting of an access token.
+In this case C and RS SHALL attempt to run the KUDOS key update protocol  {{I-D.ietf-core-oscore-key-update}} which is a lightweight alternative independent of ACE that does not require the posting of an access token. If KUDOS is not supported, then the Client and RS falls back to EDHOC as outlined above.
```

Comment:
s/independent of ACE/independent of ACE and EDHOC

Reply:
Agree